modules/sync_staging_table_with_prod.nf

```diff
@@ -0,0 +1,17 @@
+// Create data guide
+process sync_staging_table_with_prod {
+    debug true
+    container "$params.main_pipeline_docker"
+    secret 'SYNAPSE_AUTH_TOKEN'
+
+    input:
+    val previous
+
+    output:
+    stdout
+
+    script:
```

Comment:
If you ever want to do a `dry-run` for nextflow processes while still being able to go back and forth between your actual script and a `dry-run` without deleting/re-copying, this looks helpful with that: https://www.nextflow.io/docs/latest/process.html#stub

Reply:
This helps. Thanks